app/code/community/Bolt/Boltpay/Model/Payment.php

```diff
@@ -41,6 +41,7 @@ class Bolt_Boltpay_Model_Payment extends Mage_Payment_Model_Method_Abstract
     const TRANSACTION_CANCELLED = 'cancelled';
     const TRANSACTION_COMPLETED = 'completed';
     const TRANSACTION_PENDING = 'pending';
+    const TRANSACTION_ON_HOLD = 'on-hold';
```

Comment:
I think this is a status container rather than a real status

Reply:
@daisy1754 That is true.  And these transactions are all being used as "Magento order statuses" and are not exactly what the original code is implying in the name.  As the code is written, the payment class is best thought of as a status interpreter or container rather than reflecting real Bolt status.  In this light, it lacked an interpretation of the Magento 'on-hold' order status, and this as well as the other parts of the code, add this to the plugin.

Reply:
ak ok so it's not bolt's status. It is Payment status defined in Boltpay module.